src/pages/register-a-beacon/aircraft-communications.tsx

```diff
@@ -53,10 +54,14 @@ const AircraftCommunications: FunctionComponent<DraftBeaconUsePageProps> = ({
       pageHeading={pageHeading}
       showCookieBanner={showCookieBanner}
       formErrors={form.errorSummary}
-      pageText={pageText}
-      headingType="legend"
     >
-      <TypesOfCommunication form={form} />
+      <FormFieldsetAndLegend
```

Comment:
The heading should have (Optional) in as all the checkboxes are not required

https://trello.com/1/cards/6108eb640d25af412f8264f7/attachments/610a4b259bced25d98e0df66/download/Register_a_Beacon_-_DAC_Accessibility_Audit_-_300721.pdf Page 35

Reply:
changes made